internal/namespaces/rdb/v1/custom_instance.go

```diff
@@ -20,7 +20,10 @@ import (
 )
```

Comment:
Could you add a test with instance with and without public endpoint to see that it works on both cases?

Reply:
I've just added a new with public and private endpoint to test the origin of the bug. Unfortunately is not possible to test a private network connection.